docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
After discussing this on IRC, I think I'd prefer to keep the nested URL under `/accounts/`. Maybe make it `/api/v3/accounts/account/(int:user_id|string:username)/collections/collection/` (list) and `/api/v3/accounts/account/(int:user_id|string:username)/collections/collection/(int:collection_id|string:collection_slug)/` (detail) to stay consistent, but I'm not sure the `/collection/` part is really needed.

Doing it like this makes it obvious that it depends on the user account API implementation (which is good for things like user existence checks) and saves us from doing weird hacks with `lookup_value_regex` to implement the `<username>/<slug>`.